src/OrbitQt/HistogramWidget.cpp

```diff
@@ -190,6 +220,28 @@ static void DrawHint(QPainter& painter, int width) {
                     kHintSecondLineColor);
 }
 
+namespace orbit_qt {
+
+constexpr uint32_t kSeed = 31;
+
+[[nodiscard]] std::vector<int> GenerateBinWidth(size_t number_of_bins, int histogram_width) {
```

Comment:
I still think this should be `GenerateHistogramBinWidth`. Free functions require more specific names.

Reply:
Oh, I've missed "Histogram" part in your suggestion.

Reply:
Ah actually I change my suggestion to `GenerateHistogramBinWidths` (plural), as you return a vector with all the widths.

Reply:
Yeah, already done that.